docs/yaml/elements/street_segment.yaml

```diff
@@ -5,9 +5,9 @@ csv-post: |-
      :linenos:
 
 
-      id,address_direction,city,includes_all_addresses,includes_all_streets,odd_even_both,precinct_id,start_house_number,end_house_number,state,street_direction,street_name,street_suffix,unit_number,zip
-      ss000001,N,Washington,false,false,odd,pre90113,101,199,DC,NW,Delaware,St,,20001
-      ss000002,S,Washington,true,false,both,pre90112,,,DC,SE,Wisconsin,Ave,,20002
+      id,address_direction,city,includes_all_addresses,includes_all_streets,odd_even_both,precinct_id,start_house_number,end_house_number,house_number_prefix,house_number_suffix,state,street_direction,street_name,street_suffix,unit_number,zip
+      ss000001,N,Washington,false,false,odd,pre90113,101,199,DC,NW,,,Delaware,St,,20001
+      ss000002,S,Washington,true,false,both,pre90112,,,DC,SE,,,Wisconsin,Ave,,20002
```

Comment:
It looks like the additional commas put the `state` and `street_direction` fields in the new `house_number_prefix` and `house_number_suffix` fields, respectively. Translating the above into GFM Table syntax:

| id       | address_direction | city       | includes_all_addresses | includes_all_streets | odd_even_both | precinct_id | start_house_number | end_house_number | house_number_prefix | house_number_suffix | state | street_direction | street_name | street_suffix | unit_number | zip   | 
|----------|-------------------|------------|------------------------|----------------------|---------------|-------------|--------------------|------------------|---------------------|---------------------|-------|------------------|-------------|---------------|-------------|-------| 
| ss000001 | N                 | Washington | false                  | false                | odd           | pre90113    | 101                | 199              | DC                  | NW                  |       |                  | Delaware    | St            |             | 20001 | 
| ss000002 | S                 | Washington | true                   | false                | both          | pre90112    |                    |                  | DC                  | SE                  |       |                  | Wisconsin   | Ave           |             | 20002 | 

You want the following:

```
id,address_direction,city,includes_all_addresses,includes_all_streets,odd_even_both,precinct_id,start_house_number,end_house_number,house_number_prefix,house_number_suffix,state,street_direction,street_name,street_suffix,unit_number,zip
ss000001,N,Washington,false,false,odd,pre90113,101,199,,,DC,NW,Delaware,St,,20001
ss000002,S,Washington,true,false,both,pre90112,,,,,DC,SE,Wisconsin,Ave,,20002
```
Testing in GFM Table syntax:

| id       | address_direction | city       | includes_all_addresses | includes_all_streets | odd_even_both | precinct_id | start_house_number | end_house_number | house_number_prefix | house_number_suffix | state | street_direction | street_name | street_suffix | unit_number | zip   | 
|----------|-------------------|------------|------------------------|----------------------|---------------|-------------|--------------------|------------------|---------------------|---------------------|-------|------------------|-------------|---------------|-------------|-------| 
| ss000001 | N                 | Washington | false                  | false                | odd           | pre90113    | 101                | 199              |                     |                     | DC    | NW               | Delaware    | St            |             | 20001 | 
| ss000002 | S                 | Washington | true                   | false                | both          | pre90112    |                    |                  |                     |                     | DC    | SE               | Wisconsin   | Ave           |             | 20002 |

Reply:
Thank you! CSV example has been improved upon.